app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from __future__ import annotations
+
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players_list: list[Player]) -> int:
```

Comment:
Remove _list suffix from the variable name. Fix in all places
```suggestion
def calculate_team_total_rating(players: list[Player]) -> int:
```